exercises/gigasecond/.meta/hints.md

```diff
@@ -0,0 +1 @@
+If you're unsure what operations you can perform on `DateTime<Utc>` take a look at the [chrono crate](https://docs.rs/chrono/0.4.0/chrono/) which is listed as a dependency in the `Cargo.toml` file for this exercise.
```

Comment:
I wasn't sure if there was a specific format  for `.meta/hints.md` files. I looked at a couple exercism repos on Github but didn't see any. Let me know if this is alright.

Reply:
There's no specific format necessary there; it's all just human-readable text. 

You will, however, need to regenerate the README for this to pass CI:

- Get the configlet tool. On mac/linux you can just use the bin/fetch-configlet script; on Windows, you have to find it and install it manually.
- Run `bin/configlet generate . --only gigasecond` to regenerate the readme
- Commit the new README.

Reply:
Thanks for the guidance. I just updated the README.